stripe/_util.py

```diff
@@ -433,3 +433,33 @@ def _wrapper(*args, **kwargs):
                 return class_method(*args, **kwargs)
 
         return _wrapper
+
+
+def stripe_deprecate_parameter(key):
+    def stripe_deprecate_param_decorator(original_function):
+        def stripe_deprecate_param_inner(params: Mapping[str, Any], parts: List[str]):
```

Comment:
nit: you can unnest `stripe_deprecate_param_inner` and move it to the top level, since it only uses its own arguments and doesn't need `original_function` to be in scope.